<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EightQueens</class>
 <widget class="QMainWindow" name="EightQueens">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EightQueens</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ChessBoard">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>15</y>
      <width>625</width>
      <height>625</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>625</width>
      <height>625</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;br&gt;&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;br&gt;&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;br&gt;&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;br&gt;&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;br&gt;&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;br&gt;&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;br&gt;&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;
&lt;img src=&quot;:/images/Black.gif&quot;&gt;
&lt;img src=&quot;:/images/White.gif&quot;&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="HS_Popsize">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="L_Popsize">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PopSize: 1</string>
    </property>
   </widget>
   <widget class="QSlider" name="HS_MaxGen">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HS_Pc">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HS_Pm">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="L_MaxGen">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MaxGen: 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Pc">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pc: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Pm">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pm: 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Run">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="BestAnswer">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="GAParams">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;GA Parameters:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="down">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>640</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>a                 b                 c                 d                 e                 f                  g                  h</string>
    </property>
   </widget>
   <widget class="QLabel" name="left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>101</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;8&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;7&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;6&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;5&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;4&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;3&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;2&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="top">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>a                 b                 c                 d                 e                 f                  g                  h</string>
    </property>
   </widget>
   <widget class="QLabel" name="right">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>16</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;8&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;7&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;6&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;5&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;4&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;3&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;2&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="History">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>421</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;History:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
